src/list.tsx

```diff
@@ -468,8 +477,9 @@ const RowContent = React.forwardRef<TouchableElement, RowContentProps>((props, r
                 role={role}
                 dataAttributes={dataAttributes}
                 disabled={disabled}
+                aria-label={ariaLabel ?? iosVoiceOverLabelForLink}
```

Comment:
What about Android if you don't use ariaLabel prop? Does it have the same issue as iOS? If not, why not letting the Android browser handle it natively instead of setting iosVoiceOverLabelForLink?

Reply:
do you think it's worth doing platform detection for this?

Reply:
I'm not sure it will affect that much if we leave it for every platform.

Small thing: if we decide not to do it, I wouldn't name this variable "ios..."